src/components/favorite-card/favorite-card.css

```diff
@@ -0,0 +1,30 @@
+.favorite-card {
+  display: flex;
+  width: 10rem;
+  height: 10rem;
+  border-radius: 2rem;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 0.6em;
+}
+
+.favorite-card--fruits {
+  background-color: #fff2f0;
+  color: #4d0a00;
+}
+
+.favorite-card--vegetables {
+  background-color: #eff7ee;
+  color: #1c3418;
+}
+
+.favorite-card--snacks {
+  background-color: #fff8eb;
+  color: #4d3200;
+}
+
+.selection {
```

Comment:
I am not sure about the naming. I would expect that an element with the class `selection` displays something, the user already selected.

Reply:
thx I changed it to "container"